### ISSUE TITLE
CFML errors don't follow ANSI escapes

### ISSUE THREAD

**Issue by bdw429s** [Codewars/codewars-runner-cli#783]
#### Describe the bug

When an error is encountered in the CFML track, the raw ANSI escape sequences from the shell's error output are shows on the screen

```
[38;5;15m[48;5;9mERROR (4.8.0+00059)[0m
```

#### To Reproduce

Just any runtime error will do
```js
1/0;
```
#### Expected Behavior

Either implement ANSI formatting, or strip the sequences.

#### Screenshots

![image](https://user-images.githubusercontent.com/584009/73109864-0e276480-3eca-11ea-8193-da29100fe50b.png)

**Comment by kazk** [comment]
I had assumed there's no color after checking the global exception. I guess only the stack trace from the test have colors?

**Comment by kazk** [comment]
OK, divide by zero is not the cause of this colored output.

It's the test reporter crashing because of `errorStack.slice(0, 5)`, which is invalid because `5 > errorStack.len()`.

**Comment by bdw429s** [comment]
CFML Arrays start at one-- not sure if that's related or not.

**Comment by kazk** [comment]
> CFML Arrays start at one

:open_mouth:

`errorStack.slice( 0, 5 )` causes:
```
invalid call of the function arraySlice, third Argument (length) is invalid,
Offset+length cannot be greater than size of the array
```

`errorStack.slice( 1, 5 )` works.

I'm really confused by this error message. I guess something went wrong because `0` was used as `offset`.

I should be able to deploy the fix later today.

**Comment by bdw429s** [comment]
> :open_mouth:

Yep, we're one of the languages that got it _right_ :smile:   We count things like normal humans.

> I'm really confused by this error message

So about the error message-- when it says the "third" argument, that's a bald face lie.  It means the second argument.  The story behind that misleading message goes back to the history of headless and member functions I mentioned the other day.  Behind the scenes, the same implementation powers both of these:
```js
arraySlice( arr, 1, 5 )
```
and 
```js
arr.slice( 1, 5 )
```
The same validation routines were reused and the error messages clearly didn't get adjusted.  I've just entered this ticket for Lucee to address it:
https://luceeserver.atlassian.net/browse/LDEV-2657

Furthermore, second/third nonsense aside, it would appear that the `arraySlice()` function just doesn't handle an input of 0 for the offset well at all.  Zero is not a valid offset in a CFML array so the error message you received wasn't even correct.  For what it's worth, the Adobe ColdFusion engine (a separate product from Lucee) seems to hav gotten this correct, so "yay them".
https://trycf.com/gist/b161ccff2b38deea6a9f5dcc59f88372/acf2018?theme=monokai

I've entered this Lucee ticket for them to properly validate the offset as a non-zero number. 
https://luceeserver.atlassian.net/browse/LDEV-2658

And on a random side note, CFML does allow a negative offset to slice from the end of the array.
https://cfdocs.org/arrayslice
So out of the infinite number of valid positive and negative integers, you used the only whole number which was invalid :smile:
